api/migrate.ts

```diff
@@ -0,0 +1,138 @@
+import { HubSpotPromise } from '@hubspot/local-dev-lib/types/Http';
+import {
+  PLATFORM_VERSIONS,
+  UNMIGRATABLE_REASONS,
+} from '@hubspot/local-dev-lib/constants/projects';
+import { http } from '@hubspot/local-dev-lib/http';
+import { MIGRATION_STATUS } from '@hubspot/local-dev-lib/types/Migration';
+
+const MIGRATIONS_API_PATH_V2 = 'dfs/migrations/v2';
+
+interface BaseMigrationApp {
+  appId: number;
+  appName: string;
+  isMigratable: boolean;
+  migrationComponents: ListAppsMigrationComponent[];
+  projectName?: string;
+}
+
+export interface MigratableApp extends BaseMigrationApp {
```

Comment:
Should these go in the `types/` folder?

Reply:
I left them here because we don't really seem to have a hard a fast rule for putting them in the `types/` folder. We seem to have types living in files next to the methods that use them as return types, so I went this that approach since they don't really make sense as standalone types.

Reply:
That being said I don't have a strong opinion and if we want to move this to types LMK and I will move them in a follow up, going to merge this as is.